CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.5)
```

Comment:
this should be 3.8, right?

Reply:
Thanks! Well spotted! I have just updated it now.